make/test.mk

```diff
@@ -120,12 +120,12 @@ test-e2e: build build-image e2e-setup
 
 .PHONY: e2e-setup
 e2e-setup:  e2e-cleanup
-	oc new-project devconsole-e2e-test || true
+	@-oc new-project devconsole-e2e-test
 
 .PHONY: e2e-cleanup
 e2e-cleanup:
-	oc login -u system:admin
-	oc delete project devconsole-e2e-test || true
+	@-oc login -u system:admin
+	@-oc delete project devconsole-e2e-test
```

Comment:
should we maybe cleanup the other resources we created (crds, etc)?

Reply:
We will want the test to always have a "clean" starting point, so that it can be run repeatedly.

Reply:
fixed here: 36723998ea7fb932dd460587d4f242edf8029148